### ISSUE TITLE
MediaPlayer Lava Short Code Download Enabled with MP3 File But Download Link is Empty

### ISSUE THREAD

**Issue by frogydiak** [SparkDevNetwork/Rock#5426]
## Please go through all the tasks below

The download button is showing up but clicking on it will download the page HTML instead of the MP3 file and looking at the button link, you'll notice that the href is empty.

![image](https://user-images.githubusercontent.com/1373815/235326964-cf9d5f59-8c75-4c2d-92ef-3c737c5042c0.png)

## Expected Behavior
Clicking on the download button should download whatever is in the src I supposed:

![image](https://user-images.githubusercontent.com/1373815/235327115-5f090ca3-1952-4282-b65a-bcc8d735d066.png)

## Actual Behavior
Clicking on the download button will download the page HTML file instead.

## Steps to Reproduce
Use the MediaPlayer Lava short code like this:
```
{% assign sermonAudio = Item | Attribute:'SermonAudioFile','Object' %}
{% if sermonAudio and sermonAudio != empty %}
<div class="audio-player m-3">
{[ mediaplayer src:'{{ sermonAudio.Url }}' type:'audio' controls:'play,progress,mute,volume,download' ]}{[ endmediaplayer ]}
</div>
{% endif %}
```
Manually set the controls with download.

## Rock Version
14.2## Client Culture Setting
en-US

**Comment by cabal95** [comment]
@frogydiak This appears to be fixed. I would guess it was part of #5445, since that updated the Plyr library version it was most likely an issue in the library.  I tested on 15.3 and pre-alpha and both include the link correctly. I also verified that clicking on the button does actually work. I'm going to mark this as fixed in 15.3, but if you have 15.1 it is probably fixed there too.

```html
<div style="max-width: 400px;">
{[ mediaplayer src:'https://file-examples.com/storage/feaade38c1651bd01984236/2017/11/file_example_MP3_700KB.mp3' type:'audio' controls:'play,progress,mute,volume,download' ]}{[ endmediaplayer ]}
</div>
```

![image](https://github.com/SparkDevNetwork/Rock/assets/1119863/0ab6d37b-e50c-4033-a757-c08b26769ab7)

**Comment by cabal95** [comment]
@frogydiak Can you provide the actual URL that you are passing to the mediaplayer shortcode so we can replicate exactly what you are doing when the shortcode executes?

**Comment by frogydiak** [comment]
@cabal95 Here it is: https://my.lakeave.org/GetFile.ashx?guid=db4121bb-5e67-45fb-99df-675fbabfebb0

**Comment by cabal95** [comment]
@frogydiak Okay I think I was able to reproduce it. It only happens in Chrome. It seems to work fine in Firefox. Just to make sure I'm hunting down the same cause of the problem you are having, can you try in Firefox and see if the download URLs work correctly?

**Comment by frogydiak** [comment]
@cabal95 Yes, you are right, it is working properly in FireFox (Mac). I also tested it with Microsoft Edge (latest version for Mac) same thing as Chrome no file link. It is working properly with Safari though.

**Comment by cabal95** [comment]
Thanks for confirming @frogydiak. I have a fix that I will finish up Monday after some additional testing.
